.travis.yml

```diff
@@ -67,8 +57,5 @@ script:
 - make -j 2
 - ctest --verbose
 - popd
-- pushd python/tutorial
-- ${PYTHON} 101_FileIO.py
```

Comment:
Why are you not running the python tutorial anymore?

Reply:
python is gone